.github/workflows/sync-api-docs.yml

```diff
@@ -14,8 +15,62 @@ on:
       - 'openapiv2/vdp/**'
 
 jobs:
-  sync-openapi-private:
+  sync-openapi-private-dev:
+    name: Keep private (dev) docs in sync with latest pull request
```

Comment:
I see value in this (I think it'll ease the frontend review and it opens the door for our docs maintainers to be part of the review process), but my concern is that the depoloyed version won't be clear (we won't have one deployed version per PR). Is there a way we can add a reference to the PR or the commit?

Reply:
Let me check if we can add the commit hash or pr name in the doc title.

Reply:
![CleanShot 2024-07-24 at 16 37 22@2x](https://github.com/user-attachments/assets/12f9f01d-620d-452e-a599-ede366054ac2)
We can show the PR ID in the title.

Reply:
Nice 🤩 I think one version per PR will force us to upgrade our tier and that might  not be worth but this will help us to avoid confusion when reviewing the docs. 

> it opens the door for our docs maintainers to be part of the review process

Maybe this can be done only on the staging version once a PR is merged. Maybe even only once per sprint to reduce the review load.

Reply:
> it opens the door for our docs maintainers to be part of the review process
Maybe this can be done only on the staging version once a PR is merged. Maybe even only once per sprint to reduce the review load.

I think the dev document can help the engineering team with the initial review. The staging document will serve as a second round of review for all team members.